### PR TITLE
Closing changelog for 5.2.0

### PR DESCRIPTION
**Pull request by tsg**

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -14,88 +14,142 @@ https://github.com/elastic/beats/compare/v5.1.1...master[Check the HEAD diff]
...
+https://github.com/elastic/beats/compare/v5.1.1...v5.1.2[View commits]
 
-*Affecting all Beats*
+==== Bugfixes
 
-*Metricbeat*
+*Filebeat*
 
-*Packetbeat*
+- Fix registry migration issue from old states were files were only harvested after second restart. {pull}3322[3322]
```
Comment (ruflin) [inline CHANGELOG.asciidoc]: where files were (old typo)

Comment (ruflin) [inline CHANGELOG.asciidoc]: BTW: We have that now in 5.2 and 5.1.2. should we remove one entry as 5.2 is not out yet?

Comment (tsg) [inline CHANGELOG.asciidoc]: Ah, good catch about the duplicate, I forgot to do the de-duplication.

**Comment by tsg** [comment]
Added a second commit to address the comment, please squash on merge.

**Pull request merged**